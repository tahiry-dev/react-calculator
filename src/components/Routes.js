import React from 'react';
import { Switch, Route, BrowserRouter as Router } from 'react-router-dom';
import App from './App';
import Home from './Home';
import Quote from './Quote';
import NavBar from './Navbar';

const Routes = () => (
  <main>
    <Router>
      <NavBar />
      <Switch>
        <Route path="/" component={Home} exact />
        <Route path="/calculator" component={App} />
        <Route path="/quote" component={Quote} />
      </Switch>
    </Router>
  </main>
);
export default Routes;
