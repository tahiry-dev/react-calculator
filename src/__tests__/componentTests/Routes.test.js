import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Routes from '../../components/Routes';

describe('Routes', () => {
  it('renders Routes ', () => {
    const tree = renderer.create(<Routes />);
    expect(tree).toMatchSnapshot();
  });
});
