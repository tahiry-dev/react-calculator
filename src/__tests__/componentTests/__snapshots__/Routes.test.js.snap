// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes renders Routes  1`] = `
<main>
  <div>
    <div
      className="nav-container"
    >
      <div
        className="nav-left"
      >
        <h1>
          Math Magicians
        </h1>
      </div>
      <div
        className="nav-right"
      >
        <span
          className="home-link"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            Home 
          </a>
        </span>
        <span
          className="calculator-link"
        >
          <a
            href="/calculator"
            onClick={[Function]}
          >
            Calculator 
          </a>
        </span>
        <span
          className="quote-link"
        >
          <a
            href="/quote"
            onClick={[Function]}
          >
            Quote 
          </a>
        </span>
      </div>
    </div>
  </div>
  <div
    className="container"
  >
    <h2>
      Welcome to our Page!
    </h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin fringilla nulla sed commodo efficitur. Curabitur porttitor hendrerit nisl, vitae dapibus nisi imperdiet auctor. Suspendisse a orci dolor. Aliquam pharetra id ligula sit amet egestas. Maecenas tincidunt nulla turpis, eu molestie enim accumsan sit amet. In ut erat suscipit, ornare lectus quis, ullamcorper erat. Pellentesque tempus arcu ut molestie tincidunt. Ut ac leo pretium, vestibulum arcu sit amet, finibus quam. Mauris sit amet tincidunt urna. Phasellus dapibus augue vitae eleifend rhoncus.
    </p>
    <br />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin fringilla nulla sed commodo efficitur. Curabitur porttitor hendrerit nisl, vitae dapibus nisi imperdiet auctor. Suspendisse a orci dolor. Aliquam pharetra id ligula sit amet egestas. Maecenas tincidunt nulla turpis, eu molestie enim accumsan sit amet. In ut erat suscipit, ornare lectus quis, ullamcorper erat. Pellentesque tempus arcu ut molestie tincidunt. Ut ac leo pretium, vestibulum arcu sit amet, finibus quam. Mauris sit amet tincidunt urna. Phasellus dapibus augue vitae eleifend rhoncus.
    </p>
  </div>
</main>
`;
